outputs.tf

```
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
-output "bucket_name" {
-  value = "${google_storage_bucket.main.name}"
+output "names" {
+  description = "Map of name => folder resource name."
+  value       = "${zipmap(var.names, google_folder.folders.*.display_name)}"
 }
+
+output "ids" {
```

The name should indicate that this is a map rather than a list.

```suggestion
output "names_and_ids" {
```

---

Noted & changed